src/Behat/Context/UIContext.php

```diff
@@ -368,4 +368,22 @@ protected function fillFieldUnknown(NodeElement $element, $value) {
     }
   }
 
+  /**
+   * Check selector is disabled or not.
+   *
+   * @Then I check the input with :label label is disabled
```

Comment:
I suggest you a better description of the step. I am based on Behat's out of the box steps are always speculations (I should see, the url should contains...).

```suggestion
   * @Then The input with :label label should be disabled
```